<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addproduct</class>
 <widget class="QWidget" name="addproduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>471</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #19181A;
font: 600 9pt &quot;Segoe UI&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: #19181A;
color: rgb(255, 255, 255);
font: 600 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="title">
      <string>Add new</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Group</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="group_combo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>507</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Home and Kitchen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Electronics</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>26</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Product</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="model_combo">
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>17</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Product Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="productname_line">
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}

QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>64</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Brand</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="brand_line">
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}

QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>26</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="color_combo">
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yellow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pink</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Orange</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Green</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Brown</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Purple</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>White</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Black</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>50</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="number_line">
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}

QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>55</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Weight</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="weight_line">
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}

QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>66</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Price</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="price_line">
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}

QLineEdit::focus
{
	
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;

}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>22</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Warranty</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="warranty_combo">
          <property name="styleSheet">
           <string notr="true">background-color: #479761;
	color: #222629;</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>True</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>False</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>140</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirm_btn">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;</string>
          </property>
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
